airflow/www/static/js/api/useAsset.ts

```diff
@@ -27,12 +27,12 @@ interface Props {
   uri: string;
 }
 
-export default function useDataset({ uri }: Props) {
+export default function useAsset({ uri }: Props) {
   return useQuery(["dataset", uri], () => {
     const datasetUrl = getMetaValue("dataset_api").replace(
```

Comment:
```suggestion
    const datasetUrl = getMetaValue("asset_api").replace(
```
We changed the meta tag name

Reply:
This is something missed. Thanks for finding this. Just updated it